Resources/templates/CommonAdmin/ExcelAction/ExcelBuilderAction.php.twig

```diff
@@ -45,7 +45,7 @@ class ExcelController extends ListController
         $writer = $phpexcel->createWriter($phpExcelObject, '{{ builder.filetype }}');
         $response = new StreamedResponse(
             function () use ($writer) {
-                $tempFile = $this->get('kernel')->getRootDir().'/cache/' . 
+                $tempFile = '{{ builder.generator.bundleConfig.excel_tmp_dir }}/' . 
```

Comment:
@matejvelikonja no, no reason. actually that would be much simpler, I haven't thought of that :) I'll fix it :) thanks